api/v1alpha1/openapi.yaml

```diff
@@ -2426,6 +2470,22 @@ paths:
                 $ref: '#/components/schemas/Error'
 components:
   schemas:
+    DeviceDecommission:
+      type: object
+      properties:
+        decommissionType:
+          type: string
+          enum:
+            - 'DefaultUnenroll'
+          description: 'Specifies the desired decommissioning state of the device'
```

Comment:
If all we have is the default is this field even needed at this point? Have we decided what other options will exist and how they might look?

Reply:
This is meant to sketch out where we'll add those options later, once we decide exactly what they should be.